values/src/main/java/org/spine3/net/UrlPrinter.java

```diff
@@ -48,7 +48,8 @@ static String printToString(Url url) {
             return url.getRaw();
         }
 
-        //noinspection StringBufferWithoutInitialCapacity // We don't know the capacity at this point
+        //noinspection StringBufferWithoutInitialCapacity
+        // We don't know the capacity at this point
         final StringBuilder sb = new StringBuilder();
```

Comment:
In Spine we __do not__ use comment based suppression. This can be replaced since applied to a variable declaration.